src/karts/kart.cpp

```diff
@@ -1890,6 +1900,29 @@ void Kart::handleMaterialGFX(float dt)
 {
     const Material *material = getMaterial();
 
+    if (m_skidding->isSkidding())
```

Comment:
As discussed previously, this is not a good enough test for merging in master, as it is weird on many tracks.

The test should be something like this : 
`if (isOnGround() && material->isDirty())`

And you would then use skidding only to compute the rate of dirtying (it would dirty the kart faster than normal driving), speed could also be used as a factor (for reminder, 25 = normal supertux speed).